.github/workflows/test-e2e-desktop.yaml

```diff
@@ -0,0 +1,109 @@
+name: Test E2E Desktop Client
+
+on:
+  pull_request:
+    types: [labeled]
+  workflow_dispatch:
```

Comment:
- define what cadence we want this to run (every push? on PR?)
- we probably want to keep the `workflow_dispatch` option
- perhaps only run this when there are changes to the desktop client code base

Reply:
Yup, right now I just have it set to when there's a `desktop` label added which happens automatically if anything under `ui/desktop` is changed but someone could also add the label manually if for example `auth` or `core` was changed only.

I was also going to eventually add an input to the `workflow_dispatch` to take in a version for the boundary enterprise